node/malus/integrationtests/0002-dispute-invalid-block.feature

```diff
@@ -0,0 +1,23 @@
+Description: Disputes
+Network: ./0002-dispute-invalid-block.toml
+Creds: config.gcloud
+
+alice: is up
+alice: reports node_roles is 4
+alice: reports sub_libp2p_is_major_syncing is 0
+alice: reports block height is greater than 2 within 15 seconds
+alice: reports peers count is at least 2
+bob: is up
+bob: reports block height is greater than 2
+bob: reports peers count is at least 2
+charlie: is up
+charlie: reports block height is greater than 2
+charlie: reports peers count is at least 2
+david: is up
+#eve: is up
+alice: reports parachain_candidate_disputes_total is at least 1 within 225 seconds 
```

Comment:
Hi @ordian, I just used the same `wococo` config but with
```
alice:  validator
bob: malus (back-garbage-candidate)
charlie: malus (back-garbage-candidate)
david: malus (suggest-garbage-candidate)
```
And I don't get this metrics
`parachain_candidate_open_disputes`
`parachain_candidate_dispute_votes{validity="invalid"}`

I get `parachain_candidate_dispute_votes{validity="valid"}` ok, is the network configuration wrong? In the original `eve` also was set to run malus with `back-garbage-candidate`.

Thanks!